sf_examples/atari_examples/atari/atari_utils.py

```diff
@@ -38,37 +41,22 @@ def atari_env_by_name(name):
     raise Exception("Unknown Atari env")
 
 
-# noinspection PyUnusedLocal
 def make_atari_env(env_name, cfg, env_config, **kwargs):
     atari_spec = atari_env_by_name(env_name)
-
-    # TODO to render atari, need to add render_mode, will totally fix it in one week
-    # env = gym.make(atari_spec.env_id, render_mode='human')
     env = gym.make(atari_spec.env_id)
+
     if atari_spec.default_timeout is not None:
         env._max_episode_steps = atari_spec.default_timeout
 
-    assert "NoFrameskip" in env.spec.id
-
-    # if 'Montezuma' in atari_cfg.env_id or 'Pitfall' in atari_cfg.env_id:
-    #     env = AtariVisitedRoomsInfoWrapper(env)
-
-    add_channel_dim = cfg.env_framestack == 1
-    env = ResizeWrapper(
-        env,
-        ATARI_W,
-        ATARI_H,
-        grayscale=True,
-        add_channel_dim=add_channel_dim,
-        area_interpolation=False,
-    )
-
-    pixel_format = cfg.pixel_format if "pixel_format" in cfg else "HWC"
-    if pixel_format == "CHW" and add_channel_dim:
-        env = PixelFormatChwWrapper(env)
-
-    if cfg.env_framestack == 1:
-        env = SkipFramesWrapper(env, skip_frames=cfg.env_frameskip)
-    else:
-        env = SkipAndStackFramesWrapper(env, skip_frames=cfg.env_frameskip, stack_frames=4, channel_config="CHW")
+    env = gym.wrappers.RecordEpisodeStatistics(env)
+    env = NoopResetEnv(env, noop_max=30)
+    env = MaxAndSkipEnv(env, skip=cfg.env_frameskip)
+    env = EpisodicLifeEnv(env)
+    if "FIRE" in env.unwrapped.get_action_meanings():
+        env = FireResetEnv(env)
+    env = ClipRewardEnv(env)
+    env = gym.wrappers.ResizeObservation(env, (84, 84))
+    env = gym.wrappers.GrayScaleObservation(env)
+    env = FrameStack(env, cfg.env_framestack)
+    # env = gym.wrappers.FrameStack(env, 4)
```

Comment:
@wmFrank can you please check if any of the wrappers in envs/env_wrappers.py are currently not being used? In that case, just delete them :)

Reply:
ok.